spec/converters/shopify/product_converter_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+module Shopify
+  RSpec.describe ProductConverter do
+    include_context 'spree_builders'
+
+    let(:product) { build_spree_product }
+
+    describe '.initialize' do
+      subject { described_class.new(product: product) }
+
+      it "successfully does it's thing" do
+        expect(subject).to be_a described_class
+      end
+    end
+
+    describe '.to_hash' do
+      subject { described_class.new(product: product).to_hash }
+
+      it { expect(subject[:title]).to eql('name') }
```

Comment:
Are all of these values required? I ask because I am not a super fan of writing specs without descriptive strings.